src/components/PaymentHistory/ListOfPayment.tsx

```
@@ -553,6 +580,99 @@ const ListOfPayment = () => {
+          <div className="flex justify-between">
+            <p>Receipt Number</p>
+            <p>{selectedPaymentData?.report_id}</p>
+          </div>
+          <div className="flex justify-between">
+            <p>Customer Name</p>
+            <p>{selectedPaymentData?.customer_name}</p>
+          </div>
+          <div className="my-3 w-full border border-b-black border-dashed"></div>
+          <div>
+            {selectedPaymentData?.order_name.map((order: any, i: number) => (
+              <div className="flex justify-between">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)